deployments/consul-envoy/README.md

```diff
@@ -0,0 +1,79 @@
+# Monitor Consul Connect with Envoy Proxy
+
+Consul Connect emits metrics in statsd format to help understand the network. To report the metrics to SignalFx, you need to configure Consul Connect proxy to send the metrics to StatsD sink exposed by the agent.
+
+First, Configure Consul Connect Proxy to emit StatsD metrics.
+See example Consul service definition below.
+
+```hcl
+services {
+  name = "my_service"
+  port = 8080
+  connect {
+    sidecar_service {
+      proxy {
+        config {
+          envoy_extra_stats_sinks_json = <<EOF
+{
+  "name": "envoy.statsd", 
+  "config": {
+    "address": {
+      "socket_address": {
+        "address": "127.0.0.1",
+        "port_value": 8125,
+        "protocol": "UDP"
+      }
+    },
+    "prefix": "consul.my_service_mesh.my_service"
+  }
+}
+EOF
+        }
+      }
+    }
+  }
+}
+services {
+  name = "my_service2"
+  port = 9080
+  connect {
+    sidecar_service {
+      proxy {
+        config {
+          envoy_extra_stats_sinks_json = <<EOF2
+{
+  "name": "envoy.statsd", 
+  "config": {
+    "address": {
+      "socket_address": {
+        "address": "127.0.0.1",
+        "port_value": 8125,
+        "protocol": "UDP"
+      }
+    },
+    "prefix": "consul.my_service_mesh.my_service2"
+  }
+}
+EOF2
+        }
+      }
+    }
+  }
+}
+```
+
+It is mandatory to provide mesh/service name through prefix as shown since Consul does not specify the source of metric as part of the metric name. The source information will be parsed into dimensions by setting patterns in the agent configuration. For example:
```

Comment:
```suggestion
It is mandatory to provide mesh/service name through a prefix as shown since Consul does not specify the source as part of the metric name. The source information will be parsed into dimensions by setting patterns in the agent configuration. For example:
```